pajbot/utils/parse_number_from_string.py

```diff
@@ -0,0 +1,18 @@
+from typing import Union
+
+
+def parse_number_from_string(var: str) -> Union[int, float]:
+    """Tries to parse a number (int or float) from the given string.
+    If no easy conversion could be made, raise a ValueError."""
+
+    try:
+        return int(var)
+    except ValueError:
+        pass
+
+    try:
+        return float(var)
+    except ValueError:
+        pass
+
+    raise ValueError(f"could not convert string to int() with base 10 or float: {var}")
```

Comment:
```suggestion
    raise ValueError(f"could not convert string to number using int() or float(): {var}")
```

Reply:
Was initially trying to just use the phrasing that's provided by the int and float `ValueError` exceptions we are catching, which is
`ValueError: invalid literal for int() with base 10: 'asd'` and `ValueError: could not convert string to float: 'xd'` KKona
but this is nicer